Source/santactl/Commands/sync/SNTCommandSyncState.h

```diff
@@ -35,6 +35,9 @@
 /// An XSRF token to send in the headers with each request.
 @property NSString *xsrfToken;
 
+/// Full sync interval in seconds without FCM to update kDefaultFullSyncInterval
```

Comment:
`without FCM to update kDefaultFullSyncInterval` => `when FCM is not used, defaults to 10m.`